com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ProtoBuilders.java

```diff
@@ -91,12 +95,10 @@ public static String replaceTextureName(String str) {
         return out;
     }
 
-    public static String replaceTextureSetName(String str) {
-        String out = str;
-        for (String[] extReplacement : textureSetSrcExts) {
-            out = BuilderUtil.replaceExt(out, extReplacement[0], extReplacement[1]);
-        }
-        return out;
+    public static String replaceTextureSetName(String str) throws Exception {
```

Comment:
Throw an exception, so that it's easier to print where the error is coming from.